src/Our.Umbraco.BackofficeThemes/Models/ThemeInfo.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.IO;
+using Umbraco.Core;
+
+namespace Our.Umbraco.BackofficeThemes.Models
+{
+    internal class ThemeInfo
+    {
+        public string alias { get; set; }
+        public string title { get; set; }
+        public string path { get; set; }
+        public string thumbnail { get; set; }
+        public string description { get; set; }
+        public string version { get; set; }
+
+        public static ThemeInfo Parse(string file, string baseUrl)
+        {
+            if (File.Exists(file) == false)
+                return null;
+
+            var alias = Path.GetFileNameWithoutExtension(file);
+            var title = alias.ToFirstUpperInvariant();
+            var path = baseUrl + Path.GetFileName(file);
+            var thumbnail = baseUrl.Replace("/css/", "/img/") + alias + ".png";
+            string description = null, version = null;
+
+            var lines = File.ReadAllLines(file);
+            for (int i = 0; i < lines.Length; i++)
+            {
+                var line = lines[i];
+
+                // the first line must been an opening comment
+                if (i == 0 && line.StartsWith("/*") == false)
+                    return null;
+
+                // as soon as we reach a closing comment, exit
+                if (line.StartsWith("*/"))
```

Comment:
Interesting approach. We've been using .manifest JSON file for something like it in the past, but here we can definitely simplify it to this (at least for now).

Reply:
I'm not fixed on this, happy with alternative approaches.
I was purely copying the WordPress way of theme meta data.

Reply:
The longer I look at this approach the more I like it. It's really simple and based on the convention. The only concern would be with some more complex structures / themes with preprocessors etc. But still - they need to be compiled to the final stylesheet. See #7 to jump into another discussion related with the topic IMO.

I think we can leave it for now as it is and don't create problems before they really occur :)

Reply:
I've left this in for now - as I've said above, I'm happy for any alternative approaches for adding meta-data to a theme.